tests/MvcDemoApp.Tests/ExampleTest.cs

```diff
@@ -0,0 +1,10 @@
+namespace MvcDemoApp.Tests;
+
+public class ExampleTest
+{
+    [Fact]
+    public void Example_Test()
+    {
+

```

Comment:
Do you want to add at least a simple test case?

Reply:
Please see the latest commit for a real world test case of the HomeController class.